kuma/users/tests/test_views.py

```diff
@@ -1262,6 +1262,11 @@ def test_matching_accounts(self):
         assert response.status_code == 200
         assert_no_cache_header(response)
         assert not response.context['matching_accounts']
+        # The template is tested here instead of test_templates.py because
+        # test setup is so painful for login tests.
```

Comment:
I don't understand this comment. 
Do you mean "The generated HTML is tested ..."?

Also, for the record, I think it's great idea to do assertions on the `response.content` instead of doing anything with `response.context` because that would be a test of implementation details.